fibsem/milling/base.py

```diff
@@ -82,6 +89,7 @@ class FibsemMillingStage:
     strategy: MillingStrategy[Any] = field(default_factory=get_strategy)
     alignment: MillingAlignment = field(default_factory=MillingAlignment)
     imaging: ImageSettings = field(default_factory=ImageSettings) # settings for post-milling acquisition
+    reference_image: Optional[Union[FibsemImage, ImageSettings]] = None
```

Comment:
I'm not a fan of these overloaded variables, if we want to expose the alignment imaging settings to the user it should be done via adding ImageSettings to .alignment. Otherwise it's very difficult to reason about.